vtm/src/org/oscim/utils/RTree.java

```diff
@@ -419,7 +419,7 @@ public boolean search(Box bbox, SearchCb<T> cb, Object context) {
 
             while (!queue.isEmpty() && queue.peek().isLeaf) {
                 KnnItem knnItem = queue.poll();
-                T obj = (T) (knnItem.branch);
+                T obj = (T) (knnItem.branch.node);
                 results.add(obj);
                 if (results.size() >= k)
```

Comment:
According to original [source](https://github.com/mourner/rbush-knn/blob/master/index.js#L33) the comparison should be with "equals"?